front/cypress/support/commands.ts

```diff
@@ -219,26 +219,11 @@ function emailConfirmation(jwt: any) {
   });
 }
 
-function completeRegistration(jwt: any) {
-  return cy.request({
-    headers: {
-      'Content-Type': 'application/json',
-      Authorization: `Bearer ${jwt}`,
-    },
-    method: 'POST',
-    url: 'web_api/v1/users/complete_registration',
-    body: {
```

Comment:
This endpoint doesn't exist anymore, but we don't seem to need this for our e2e tests either :+1: